### ISSUE TITLE
Globstar Not Detecting Root Directory Files

### ISSUE THREAD

**Issue by iansantagata** [linthtml/linthtml#411]
### Problem
When running `linthtml **/*.html`, HTML files in the root directory are not reported as found and are thus not analyzed or linted.

There seems to be a bug or unexpected feature that is not explained with which files are ignored and in what directories.

Note that this is with no configuration file.

### Steps to Reproduce

1. Setup a directory and file structure like the following:

```
rootProjectDirectory
|---> rootTest.html
|---> nestedDirectory
        |---> nestedTest.html
```

2. Run `linthtml **/*.html` when the current directory is `rootProjectDirectory`.
3. Observe output of "Found 1 files" when 2 is expected.

### Work-Arounds
It's possible to get the current directory and all nested directories with the command `linthtml "(**/*.html|*.html)"`, but this is unwieldy and the globstar character `**` should already handle that case.

### Details

When running Globby on a similar glob like the following, root directory HTML files are found.

```js
const globby = require("globby");

const result = globby.sync(["**/*.html", "!node_modules/"], {
    gitignore: true,
    expandDirectories: {
        files: ["**/*.html"],
        extensions: ["html"]
    }
});
```

This code is copied from the code here to reach out to Globby - https://github.com/linthtml/linthtml/blob/main/lib/index.js#L59-L66

It makes the possibility that the issue is upstream within Globby highly unlikely as well.

**Comment by KamiKillertO** [comment]
@iansantagata I've run some tests on Linux and everything seems to be working just fine.

![image](https://user-images.githubusercontent.com/9579729/139901610-d9687e0c-adae-4b2d-abfe-2a0ad1d6223d.png)

The documentation about the ignore is somehow lacking some explanations, here's how the feature works.
When running `linthtml pattern` : 
* the cli will start by looking if there's a `.linthtmlignore` file in the current folder
  * If there's a `.linthtmlignore` file, it will use it to know the files to ignore.
  * If there's no `.linthtmlignore` file, then the CLI will fallback to the `.gitignore` file to get the list of files to ignore unless the pattern provided is an absolute path

**Comment by iansantagata** [comment]
Ah, I see what the problem here is actually.  I have duplicated your example myself in a new folder and created a `package.json` class at root in that folder like this:

```json
{
  "name": "Test",
  "version": "1.0.0",
  "main": "index.js",
  "devDependencies": {
    "@linthtml/linthtml": "^0.8.0"
  },
  "scripts": {
    "linthtml": "linthtml **/*.html"
  }
}
```

If I am in `test-folder` and I run `npm run linthtml`, which will call the script in the `scripts` section of the `package.json` file, it will pick up all 4 files and analyze them properly with the globstar.

However, when I run `npx linthtml **/*.html` manually in the same folder, it only finds 2 files.  After deleting some of them and testing, it looks like it only finds the ones in `nestedFolder`, but not any in the project's root directory (here is `test-folder`) or in `deepNested`.  That seems somewhat problematic to me, although I'm not sure why the two would be different.

For reference, and in case it matters, I'm running on Windows 7 and running the commands listed above in Git Bash for Windows.  I don't have any `.linthtmlignore` or `.gitignore` files in my test directory.

Screenshots below:

![image](https://user-images.githubusercontent.com/7070901/139912293-848a1f45-dd2f-4dfc-8b1b-bd74b59a0bd6.png)

![image](https://user-images.githubusercontent.com/7070901/139912226-1e267f19-53a9-4836-a821-3876f1efc7b6.png)

![image](https://user-images.githubusercontent.com/7070901/139912874-adf385b8-e7e1-4b5f-bfc5-fb2dd3e22c74.png)

**Comment by iansantagata** [comment]
Any ideas why these two would be different?  I would expect the output of both of them to be the same.

**Comment by KamiKillertO** [comment]
Ok, I was not able to reproduce your problem using npx but I was able to reproduce it with the npm script.
I'm not sure why you are having the issue with npx but what I can tell you is that you should use quotes around glob patterns.
It's explained in details in this [article](https://medium.com/@jakubsynowiec/you-should-always-quote-your-globs-in-npm-scripts-621887a2a784)
Everything should be fine if you run `npx linthtml '**/*.html'`

**Comment by iansantagata** [comment]
Ah, that might be the problem!  Thanks for the debugging help here!

Definitely seems more like user error at this point than anything to do with the library, so closing this issue.

Thanks again!

**Comment by Kristinita** [comment]
**Type: Possible reason of the problem**

### 1. Quotes

The problem can also arise because linthtml requires "double quotes". 'Single quotes' don’t work.

### 2. Example

```text
D:\Kristinita>linthtml 'KiraFolder/*.html'
√ Found 0 files
√ Files analyzed

✨  There's no problem, good job �
```

```
D:\Kristinita>linthtml "KiraFolder/*.html"
√ Found 9 files
√ Files analyzed

✨  There's no problem, good job �
```

### 3. Environment

1. Microsoft Windows [Version 10.0.19041.1415]
2. linthtml 0.8.6

Thanks.
